[2024-04-20T16:08:01.185+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-20T16:08:01.338+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2024-04-17T00:00:00+00:00 [queued]>
[2024-04-20T16:08:01.395+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2024-04-17T00:00:00+00:00 [queued]>
[2024-04-20T16:08:01.399+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-20T16:08:01.476+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): third_task> on 2024-04-17 00:00:00+00:00
[2024-04-20T16:08:01.577+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2384) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-20T16:08:01.604+0000] {standard_task_runner.py:63} INFO - Started process 2415 to run task
[2024-04-20T16:08:01.644+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'third_task', 'scheduled__2024-04-17T00:00:00+00:00', '--job-id', '311', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpswkzgkdp']
[2024-04-20T16:08:01.691+0000] {standard_task_runner.py:91} INFO - Job 311: Subtask third_task
[2024-04-20T16:08:02.082+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v3.third_task scheduled__2024-04-17T00:00:00+00:00 [running]> on host e87e2f3305bd
[2024-04-20T16:08:02.340+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='julionevadod' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-17T00:00:00+00:00'
[2024-04-20T16:08:02.342+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-20T16:08:02.387+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-20T16:08:02.393+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task3 and will be running after task 1']
[2024-04-20T16:08:02.414+0000] {subprocess.py:86} INFO - Output:
[2024-04-20T16:08:02.416+0000] {subprocess.py:93} INFO - hey, I am task3 and will be running after task 1
[2024-04-20T16:08:02.420+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-20T16:08:02.431+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-20T16:08:02.482+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=third_task, execution_date=20240417T000000, start_date=20240420T160801, end_date=20240420T160802
[2024-04-20T16:08:02.540+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-20T16:08:02.586+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-20T16:08:02.589+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
